outport/factory/outportFactory.go

```diff
@@ -39,9 +41,30 @@ func createAndSubscribeDrivers(outport outport.OutportHandler, args *OutportFact
 		return err
 	}
 
+	err = createAndSubscribeCovalentDriverIfNeed(outport, args.CovalentIndexerFactoryArgs)
+	if err != nil {
+		return err
+	}
+
 	return nil
 }
 
+func createAndSubscribeCovalentDriverIfNeed(
```

Comment:
unit tests?

Reply:
I've added:
- A unit test to subscribe covalent indexer 
- Some unit tests for each driver subscription, which also increased code coverage with ~40%. (currently ~98%)

However, I did create a test to successfully subscribe an elastic indexer